<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Field Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <property name="spacing">
     <number>12</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="info_label">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>B0 viewer</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QCheckBox" name="XYcheckBox">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>SLICE XY</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="XYspinBox">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="ZXcheckBox">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>SLICE ZX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ZXspinBox">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="YZcheckBox">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>SLICE YZ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="YZspinBox">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QCheckBox" name="ShowSphereCheckBox">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>sphere</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ShowMagnetCheckBox">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string> magnet</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ShowRingsCheckBox">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>rings</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QCheckBox" name="PlotRawCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Raw</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PlotSPHCheckBox">
            <property name="text">
             <string>SPH</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PlotErrorFieldCheckBox">
            <property name="text">
             <string>Total</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QCheckBox" name="PlotShimFieldCheckBox">
            <property name="text">
             <string>fair shim</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PlotCheapFieldCheckBox">
            <property name="text">
             <string>cheap shim</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="show_2d_slice_btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>show 2d slice</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>                    before   |   after</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mean_field_label">
          <property name="text">
           <string>Mean init: N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="inhomogeneity_label">
          <property name="text">
           <string>Homo init: N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mean_field_shimmed_label">
          <property name="text">
           <string>Mean After: N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="inhomogeneity_after_label">
          <property name="text">
           <string>Homo After: N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Harmonic</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sph_spinbox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>15</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Resolution [mm]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="resolution_SPH_spinbox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>250</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="SPH_button">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>SPH DECOMP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>DSV for opt [%]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="dsv_opt_spinbox">
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="verbose_checkbox">
          <property name="text">
           <string>verbose</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="get_shim_positions_btn">
          <property name="text">
           <string>CALCULATE SHIM ROTATIONS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_rings_button">
          <property name="text">
           <string>Save Rings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_rings_button">
          <property name="text">
           <string>Load Rings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="load_csv_button">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Load Field Map</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="coordinate_transform_btn">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>To Magnet Coordinates</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>generate synthetic CSV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="export_for_Tom_btn">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>export separately</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_CV_plotter">
          <item>
           <widget class="QWidget" name="plotter_widget" native="true">
            <property name="minimumSize">
             <size>
              <width>420</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1080</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2d_plotter">
      <item>
       <widget class="QWidget" name="plotter_widget_2d" native="true">
        <property name="minimumSize">
         <size>
          <width>420</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1080</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
